.changeset/cuddly-insects-pump.md

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
Note: the feature changset for email link verification will be in another PR. This PR is just for the breaking changes and using the new JS autoSignIn feature.